src/ConfigManager.cpp

```diff
@@ -303,6 +331,19 @@ void ConfigManager::startApi() {
     server->begin();
 }
 
+void ConfigManager::baseRoutingSetup() {
```

Comment:
How about calling this `createBaseWebServer` to make it a bit clearer that it creates a new web server.